developer-docs-site/docs/tutorials/your-first-nft.md

```diff
@@ -58,7 +58,7 @@ git clone git@github.com:aptos-labs/aptos-core.git ~/aptos-core
   poetry update
   ```
 
-  Run the `transfer-coin` example:
+  Run the `simple_nft` example:
```

Comment:
nit simple-nft

Reply:
Oops, it's already auto-merged. I will submit another PR.